pkg/webhooks/mutation.go

```diff
@@ -150,9 +150,11 @@ func (ws *WebhookServer) applyMutation(request *v1beta1.AdmissionRequest, policy
 		return nil, nil, fmt.Errorf("failed to apply policy %s rules %v", policyContext.Policy.Name, engineResponse.GetFailedRules())
 	}
 
-	err := ws.openAPIController.ValidateResource(*engineResponse.PatchedResource.DeepCopy(), engineResponse.PatchedResource.GetAPIVersion(), engineResponse.PatchedResource.GetKind())
-	if err != nil {
-		return nil, nil, errors.Wrapf(err, "failed to validate resource mutated by policy %s", policyContext.Policy.Name)
+	if engineResponse.PatchedResource.GetKind() != "*" {
```

Comment:
What about validate policies? Should we also skip validating?

Reply:
@realshuting this is check for OpenApi check for ValidateResource. For validate policy don't have that check.
we have on more check here, we already covered.
https://github.com/kyverno/kyverno/blob/b91ff5a7f213ca1a9d0aedf02773070c893a9823/pkg/openapi/validation.go#L167